config/config_test.go

```diff
@@ -103,6 +150,23 @@ func TestConfigS3_IncludeNames(t *testing.T) {
 	return
 }
 
+func TestConfigIamRole_IncludeNames(t *testing.T) {
+	configFilePath := "./mocks/iamRole_include_names.yaml"
+	configObj, err := GetConfig(configFilePath)
+
+	require.NoError(t, err)
+
+	if reflect.DeepEqual(configObj, emptyConfig()) {
+		assert.Fail(t, "Config should not be empty, %+v\n", configObj)
+	}
+
+	if len(configObj.IamRole.IncludeRule.NamesRE) == 0 {
+		assert.Fail(t, "ConfigObj should contain IAM Role names regexes, %+v\n", configObj)
+	}
```

Comment:
NIT: this if block can be reduced to `assert.Greater(t, len(configObj.IamRole.IncludeRule.NamesRE), 0)`

This applies to similar calls in other tests.

Reply:
Leaving this one as TODO.